chef/roles/dns-server.rb

```diff
@@ -2,7 +2,7 @@
 name "dns-server"
 description "DNS Server Role - DNS server for the cloud"
 run_list(
-         "recipe[bind9]"
+         "recipe[dns::role_dns_server]"
```

Comment:
Style/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.